### ISSUE TITLE
Imp Ignorance Rung Clarification 

### ISSUE THREAD

**Issue by Jkosio23** [mraof/Minestuck#471]
https://github.com/mraof/Minestuck/blob/fd80e52a3abc490bd3fed8012cb0e3437a73b88b/src/main/java/com/mraof/minestuck/entity/underling/ImpEntity.java#L112

if the requirement for imps ignoring you is "_reach_ rung x" then this line of code should say 			

return PlayerSavedData.getData((ServerPlayer) entity).getEcheladder().getRung() < x;

not 
return PlayerSavedData.getData((ServerPlayer) entity).getEcheladder().getRung() < x+1;

**Comment by Jkosio23** [comment]
because the list of rungs starts at 0 instead of 1

**Comment by kirderf1** [comment]
Yeah, I see the point. So with the code as it is right now, you actually need to reach rung at index 16 aka the 17th rung.

With that said, what would people want it changed to? Should it be kept at 16 with just the incorrect comment removed, be changed to the rung at index 15, or maybe make it the 15th rung instead? (or even lower?)
Regardless, changing it to be a named constant at the top of the file should be appropriate.

**Comment by Jkosio23** [comment]
at what rank does xp from imps become inconsequential? because if it's lower than 15 then i'd say that rung, but if it's higher then 15 does sound good
